app/src/examples/AnimatedSensorAccelerometerExample.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Pressable, SafeAreaView, StyleSheet, Text, View } from 'react-native';
+import Animated, {
+  SensorType,
+  useAnimatedSensor,
+  useAnimatedStyle,
+  useSharedValue,
+  withSpring,
+} from 'react-native-reanimated';
+
+function clampWorklet(num: number, min: number, max: number) {
```

Comment:
This worklet is very useful, let's rename this function to `clamp` and add it to public interface of Reanimated (either in this or in a separate PR 🙏 )

```suggestion
function clamp(num: number, min: number, max: number) {
```

also cc @tjzel who said no one's gonna need it

Reply:
no one is gonna use it 😼

Reply:
I will prepare a separate PR and we can discuss it there